install.md

```diff
@@ -1,4 +1,4 @@
-# Ubuntu installation instructions
+# Installation instructions.
```

Comment:
we can do that and mention on the line with the ubuntu install script that this line is only for ubuntu, otherwise we assume you've installed postgres already, that script is the only reason to keep ubuntu in the title